src/App.tsx

```diff
@@ -11,6 +11,25 @@ import { CoreOrganiserProvider } from './datastore/contexts/CoreOrganiserProvide
 import { TTWireSourceProvider } from './datastore/contexts/TTWireSourceProvider'
 import { ModalProvider } from './components/Modal/ModalProvider'
 import { LoadingText } from './components/LoadingText'
+import {
+  getWebInstrumentations,
+  initializeFaro,
+  ReactIntegration
+} from '@grafana/faro-react'
+
+const FARO_URL = (import.meta.env.VITE_FARO_URL || '') as string
+const FARO_NAME = (import.meta.env.VITE_FARO_NAME || '') as string
+
+initializeFaro({
+  url: FARO_URL,
```

Comment:
In the browser console, when the page loads, I get
```
App.tsx:23 Faro
 either "url" or "transports" must be defined
```

Maybe a conditional check here would suffice?

Reply:
Shouldn't you get a warning if a necessary env is missing?